learn/tooling-guide/cli-tools/openapi.md

```diff
@@ -23,36 +23,32 @@ The OpenAPI tools provide the following capabilities.
  2. Export the OpenAPI definition of a Ballerina service.
  3. Validate service implementation of a given OpenAPI Contract.
     
-The `openapi` command in Ballerina is used for OpenAPI to Ballerina and Ballerina to OpenAPI code generations. 
-Code generation from OpenAPI to Ballerina can produce `ballerina service stubs` and `ballerina client stubs`.
-The OpenAPI compiler plugin will allow you to validate a service implementation against an OpenAPI contract during
- compile time. 
-This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.   
+The `openapi` command in Ballerina is used for OpenAPI to Ballerina and Ballerina to OpenAPI code generations. Code generation from OpenAPI to Ballerina can produce `ballerina service stubs` and `ballerina client stubs`.
+
+The OpenAPI compiler plugin will allow you to validate a service implementation against an OpenAPI contract during the compile time. This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.   
 
 ### OpenAPI to Ballerina
-#### Generate Service and Client Stub from an OpenAPI Contract
+
+#### Generating Service and Client Stub from an OpenAPI Contract
 
 ```bash
 bal openapi -i <openapi-contract-path> 
-               [--service-name: generated files name]
                [--tags: tags list]
                [--operations: operationsID list]
                [--mode service|client ]
                [(-o|--output): output file path]
 ```
-Generates both the Ballerina service and Ballerina client stub for a given OpenAPI file.
 
-This `-i <openapi-contract-path>` parameter of the command is mandatory. As an input, it will take the path to the
- OpenAPI contract file (i.e., `my-api.yaml` or `my-api.json`).
+Generates both the Ballerina service and Ballerina client stub for a given OpenAPI file.
 
-The `--service-name`  is an optional parameter, which allows you to change the generated service name.
+This `-i <openapi-contract-path>` parameter of the command is mandatory. As an input, it will take the path to the OpenAPI contract file (i.e., `my-api.yaml` or `my-api.json`).
 
 You can give the specific tags and operations that you need to document as services without documenting all the operations using these optional `--tags` and `--operations` commands.
 
-`(-o|--output)` is an optional parameter. You can use this to give the output path for the generated files.
-If not, it will take the execution path as the output path.
+`(-o|--output)` is an optional parameter. You can use this to give the output path for the generated files. If not, it will take the execution path as the output path.
```

Comment:
@lnash94 What's meant by "execution path" here? The current directory?

Reply:
yeah, it means the current directory path

Reply:
Changed to: "If not, it will take the path of the current directory as the output path"